src/lib/components/FileUploader/FileUploader.js

```diff
@@ -66,10 +68,37 @@ export const FileUploaderComponent = ({
       );
       const maxFileStorageReached =
         filesSize + acceptedFilesSize > quota.maxStorage;
-      if (maxFileNumberReached || maxFileStorageReached) {
-        // TODO: Give some feedback to user about why the files weren't
-        // accepted for upload
-        console.log('Maximum number of files reached!');
+
+      if (maxFileNumberReached) {
+        setWarningMsg(
+          <div className="content">
+            <Message
+              warning
+              icon="warning circle"
+              header="Could not upload files."
+              content={`Uploading the selected files would result in ${
+                filesList.length + acceptedFiles.length
+              } files (max.${quota.maxFiles})`}
+            />
+          </div>
+        );
+      } else if (maxFileStorageReached) {
+        setWarningMsg(
+          <div className="content">
+            <Message
+              warning
+              icon="warning circle"
+              header="Could not upload file(s)."
+              content={
+                <>
+                  Uploading the selected files would result in{' '}
+                  {humanReadableBytes(filesSize + acceptedFilesSize)} bytes and
+                  the limit is {humanReadableBytes(quota.maxStorage)} bytes.
```

Comment:
minor: `humanReadableBytes` already gives the result in bytes, so we should be able to skip `bytes` : 
 
```suggestion
                  {humanReadableBytes(filesSize + acceptedFilesSize)}, but
                  the limit is {humanReadableBytes(quota.maxStorage)}.
```